keras_contrib/applications/densenet.py

```diff
@@ -106,7 +202,7 @@ def DenseNet(input_shape=None, depth=40, nb_dense_block=3, growth_rate=12, nb_fi
                                       default_size=32,
                                       min_size=8,
```

Comment:
does min_size and default_size change for imagenet vs cifar weights?

Reply:
I was able to load both sets of weights for all 3 models with no alterations to this, so I didn't set specific min size and default size. Seems as long as an input size is provided, there is no issue with this.